README.md

```diff
@@ -3,23 +3,23 @@
 **A site displaying public IPFS gateways and their online/offline status.**
 
 View the Public Gateway Checker in action
-* on Fleek:  https://ipfs-public-gateway-checker.on.fleek.co/
-* on GitHub Pages: https://ipfs.github.io/public-gateway-checker/
+
+- on Fleek: <https://ipfs-public-gateway-checker.on.fleek.co/>
+- on GitHub Pages: <https://ipfs.github.io/public-gateway-checker/>
 
 [![Screenshot of Public Gateway Checker](https://user-images.githubusercontent.com/157609/121263486-f7fb2800-c8b5-11eb-9061-0b6f586a6f25.png)](https://ipfs.github.io/public-gateway-checker/)
 
 ## SECURITY NOTES
 
--   The list contains gateways operated by various parties, coordinated by loose mutual consensus, without a central governing authority. Protocol Labs operates and is responsible for only two of the listed gateways: `ipfs.io` and `dweb.link`.
--   Gateways without origin isolation will be marked with ⚠️, indicating they are not safe for use cases that require private local storage of data or credentials. [Learn more](https://github.com/ipfs/public-gateway-checker/issues/150).
+- The list contains gateways operated by various parties, coordinated by loose mutual consensus, without a central governing authority. Protocol Labs operates and is responsible for only two of the listed gateways: `ipfs.io` and `dweb.link`.
+- Gateways without origin isolation will be marked with ⚠️, indicating they are not safe for use cases that require private local storage of data or credentials. [Learn more](https://github.com/ipfs/public-gateway-checker/issues/150).
 
 ## Adding a new public gateway
 
-If you'd like to add a new public gateway, please edit `./gateways.txt`:
+If you'd like to add a new public gateway, please edit `./gateways.json`:
 
 1. Add the gateway's address to the bottom of the list
```

Comment:
This will avoid diff on two lines due to trailing `,`
```suggestion
1. Add the gateway's address to the **top** of the list
```

Reply:
Thinking ahead!